extension.js

```diff
@@ -313,9 +317,50 @@ function workarea_for_window(win) {
 }
 
 function target_rect_for_workarea(workarea) {
-    const target_rect = workarea.copy();
-    target_rect.height *= settings.get_double('window-height');
-    return target_rect;
+    const available_width = workarea.width,
+        available_height = workarea.height,
```

Comment:
What's the point of adding `available_width` and `available_height`? `workarea.width` and `workarea.height` are even shorter by one character.